### PR TITLE
19203 Configure stylesheet loading to enable third-party component styling

### PR DESCRIPTION
**Pull request by myabc**
:warning: ~~**Includes commits from #2695, #2698. Please review, merge first.**~~

https://community.openproject.org/work_packages/19203

Relates to:
- https://community.openproject.org/work_packages/18090
- https://community.openproject.org/work_packages/17811

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://myabc-feature-19203-enable-3rd-party-component-styling.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-feature-19203-enable-3rd-party-component-styling.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by myabc** [comment]
This is working for jQuery UI with Gulp. The asset pipeline will take more work…

**Comment by myabc** [comment]
@ulferts @floriank ready for review. As noted in the daily: _this is only working when the style guide is built with gulp_. Only gulp-livingstyleguide allows for the entire style guide template to be easily customised to the extent we need: additional style, script tags + `ng-app` on `body`.

File: app/assets/javascripts/application.js.erb [hunk 1]
```diff
@@ -67,48 +67,6 @@ jQuery(document).ready(function ($) {
...
-      }
-    };
-
-    if (CS.firstWeekDay && CS.firstWeekDay !== "") {
-      defaults.firstDay = parseInt(CS.firstWeekDay, 10);
-    }
-
-    $.datepicker.setDefaults(defaults);
-  }
-
```
Comment (floriank) [inline app/assets/javascripts/application.js.erb:111]: :+1: Removing stuff from `application.js.erb`

**Pull request merged**